pages/index.tsx

```diff
@@ -31,7 +31,7 @@ export default function Home({ claimData, loading }: HomeProps): ReactElement {
     <Container fluid className="index">
       <Head>
         <title>{t('title')}</title>
-        <link rel="icon" href="/favicon.ico" />
+        <link rel="icon" href="claimstatus/favicon.ico" />
```

Comment:
I don't think it matters in this case since we're at the root it'll resolve the same, but I'll put it as absolute to be consistent
```suggestion
        <link rel="icon" href="/claimstatus/favicon.ico" />
```